src/libraries/System.Private.CoreLib/src/System/EventHandler.cs

```diff
@@ -5,5 +5,10 @@ namespace System
 {
     public delegate void EventHandler(object? sender, EventArgs e);
 
-    public delegate void EventHandler<TEventArgs>(object? sender, TEventArgs e); // Removed TEventArgs constraint post-.NET 4
+    public delegate void EventHandler<in TEventArgs>(object? sender, TEventArgs e) // Removed TEventArgs constraint post-.NET 4
+        where TEventArgs : allows ref struct;
+
+    public delegate void EventHandler<in TSender, in TEventArgs>(TSender sender, TEventArgs e)
```

Comment:
@tannergooding how do I add XML docs for APIs in old locations? Was there a separate repo?

Reply:
Add the XML comment here.

Reply:
Oh that's convenient, done! Thank you 🙂